lib/recurly/add_on.rb

```diff
@@ -21,7 +22,7 @@ class AddOn < Resource
       created_at
       updated_at
       tier_type
-      tiers
+      # tiers
```

Comment:
This results in a behavior I don't think you expected:

```
irb(main):001:0> x = %w(
irb(main):002:0]   one
irb(main):003:0]   two
irb(main):004:0]   # three)
=> ["one", "two", "#", "three"]
```

Reply:
nope absolutely not!